omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -124,6 +124,13 @@
     python = "#{install_dir}/embedded/bin/python3"
   end
 
+  # If a python_mirror was set, it's passed through a pip config file so that we're not leaking the API key in the CI Output
+  # Else the pip config file so pip will act casually
```

Comment:
```suggestion
  # If a python_mirror is set, it is set in a pip config file so that we do not leak the token in the CI output```

Reply:
Right, fixed in [a3141ba](https://github.com/DataDog/datadog-agent/pull/14473/commits/a3141bad2d9ae6adc59125ab2c311dcd95b86299) and [98c297a](https://github.com/DataDog/datadog-agent/pull/14473/commits/98c297a956f8c0a8919621fa495d3bcc214107a2) !